odoo_tools/cli/release.py

```diff
@@ -4,12 +4,24 @@
 import click
 
 from ..config import get_conf_key
+from ..utils.git import get_current_branch
 from ..utils.marabunta import MarabuntaFileHandler
 from ..utils.misc import get_ini_cfg_key
 from ..utils.os_exec import run
 from ..utils.path import build_path
 from ..utils.pending_merge import push_branches
 
+END_TIPS = [
+    "Please continue with the release by:",
+    " * Cleaning HISTORY.rst. Remove the empty sections, empty lines...",
```

Comment:
This is not needed. The changelog will be automatically generated w/ files in `changes.d`